src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -220,6 +220,15 @@ public object ExplicitDefaultValue
         }
 
         /// <summary>
+        /// Returns the default value expression of the parameter,
+        /// or null if the parameter doesn't have a default value or
+        /// the parameter type is a struct and the default value of the parameter
+        /// is the default value of the struct type or of type parameter type which is
+        /// not known to be a referenced type.
+        /// </summary>
```

Comment:
>/// </summary> [](start = 8, length = 14)

This sounds too complicated. I would go with a simple contract - if there is syntax for the default value, you get BoundExpression for it, otherwise null.